helmchart.yaml

```diff
@@ -0,0 +1,7212 @@
+apiVersion: v1
```

Comment:
please remove this file, it should not be part of this PR

Reply:
Alright I will do that.